CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 Added:
 
-- Emoji picker accessible via `:` in text input.  See [configuration](https://halloy.squidowl.org/configuration/buffer.html#bufferemojis)
+- Emoji picker accessible via `:` in text input and emoji auto-replace for `:shortcode:`.  See [configuration](https://halloy.squidowl.org/configuration/buffer.html#bufferemojis)
```

Comment:
Nit pick. Going forward ill just have a link to configuration on top of the CHANGELOG so we dont have to write "See configuration" on every line. I would just rewrite this to:

> - Emoji picker accessible via `:` in text input
> - Auto conversion from :shortcode: to the corresponding emoji